src/docs/Updates/Roadmap/index.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Roadmap'
+order: 3
+description: 'Discover what we are working on right now, and how you could contribute to Mozaic'
+---
+
+## Q1 2020 (from 01/01/2020 to 01/04/2020)
+
+#### Create a bêta version of the Mozaic-vue library
+
+- Implement CI / CD and code quality tools
+- Create contributing and coding guidelines
+- Create and migrate at least all the forms components existing in Mozaic into Mozaic-vue
+- Link Mozaic-vue in the Mozaic website
+- Support contribution from other teams
+
+#### Core team components migration into Mozaic
+
+- Help the core team migrate existing components into mozaic
+  - popin / modal component
+  - visibility utilities (for responive and screen readers)
+  - reset / a css file to reset and define basic global styles
+  - images / videos components
+  - separators / dividers component
+  - wysiwyg scope styles (styles required for wysiwyg output)
+
+#### Support Core team into migrating Mozaic into common socle
+
+- Implement mozaic components as freemarker templates
+- Distribute larger parts of Mozaic through common socle
```

Comment:
Perhaps these parts can be grouped into one section for clarity. Something like:

### Working closely with the Core Team
- Help the core team migrate existing components into mozaic
  - popin / modal component
  - visibility utilities (for responive and screen readers)
  - reset / a css file to reset and define basic global styles
  - images / videos components
  - separators / dividers component
  - ~~wysiwyg scope styles (styles required for wysiwyg output)~~ <= **It won't migrate into Mozaic. It's a core team element.**

- Support of the core team during the generalization of the use of Mozaic
  - Distribute main parts of Mozaic through common socle
  - Mozaic components as freemarker templates
  - Mozaic components in Kobi